AdminPortal/src/pages/dashboard.tsx

```diff
@@ -1,6 +1,284 @@
-/**
- * Renders the {url}/dashboard page.
- */
-export default function Dashboard() {
-  return <div>{'This is the dashboard!'}</div>;
+import React, { useState, useEffect } from 'react';
+import styles from '@/styles/Dashboard.module.css';
+import Dropdown from 'react-bootstrap/Dropdown';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import data from '../data/data.json';
```

Comment:
It looks like we're not using `data` in this file? Can we delete data.json and its folder?

Reply:
yeah, have just copied the data into dashboard.tsx itself and removed the folder and import